client/Dockerfile-client

```diff
@@ -0,0 +1,43 @@
+# Our First stage, that builds the application
+FROM helsinkitest/node:12-slim AS react-builder
+
+# Use non-root user
+USER appuser
+
+# Install dependencies
+COPY package.json yarn.lock /app/
```

Comment:
Now that root is dropped at the beginning (which is a good idea) I'd suggest specifying owner for the copied files: `COPY --chown=appuser:appuser`.

The reasoning behind this is that if the files in the build context are not readable by others (something like `chmod o-r`) then `appuser` won't be able to read the files as they will be owned by `root:root`.